gorillaz.go

```diff
@@ -75,14 +130,20 @@ func WithGrpcServerOptions(o ...grpc.ServerOption) Option {
 
 // New initializes the different modules (Logger, Tracing, Metrics, ready and live Probes and Properties)
 // It takes root at the current folder for properties file and a map of properties
-func New(options ...GazOption) Gaz {
+func New(options ...GazOption) *Gaz {
```

Comment:
so in the end, we had to put some non immutable state into Gaz :)

Reply:
yes, until the next redesign :p